### PR TITLE
GNOME - Translate desktop file and metainfo using strings from *.resx

### PR DESCRIPTION
**Pull request by fsobolev**
A new python script is added. It loops over `Strings.*.resx` files, gets translations for `MetaDescription.GTK` and adds new strings in proper formats to desktop and metainfo files. The script accepts install prefix as an argument, exactly like `install_extras.sh`, which calls this script now.
I copied strings from old desktop file to existing `Strings.*.resx`, and now after installation desktop and metainfo files have the following contents:

```
[Desktop Entry]
Version=1.0
Name=Money
Comment=A personal finance manager
Comment[es]=Un gestor de finanzas personales
Comment[fr]=Gestionnaire des dépenses personnelles
Comment[hi]=एक व्यक्तिगत वित्त संयोजक
Comment[hr]=Aplikacija za osobne financije
Comment[id]=Manajer keuangan pribadi
Comment[it]=Gestione finanziaria personale
Comment[ru]=Персональный менеджер финансов
Exec=/app/opt/org.nickvision.money/NickvisionMoney.GNOME
Icon=org.nickvision.money
Terminal=false
Type=Application
MimeType=application/x-nmoney
X-GNOME-UsesNotifications=true
X-Desktop-File-Install-Version=0.26
```

```
<?xml version="1.0" encoding="UTF-8"?>
<component type="desktop-application">
  <id>org.nickvision.money</id>
  <metadata_license>CC0-1.0</metadata_license>
  <project_license>GPL-3.0+</project_license>
  <name>Money</name>
  <summary>A personal finance manager</summary>
  <description>
    <p>
      A personal finance manager
    </p>
    <p xml:lang="es">
      Un gestor de finanzas personales
    </p>
    <p xml:lang="fr">
      Gestionnaire des dépenses personnelles
    </p>
    <p xml:lang="hi">
      एक व्यक्तिगत वित्त संयोजक
    </p>
    <p xml:lang="hr">
      Aplikacija za osobne financije
    </p>
    <p xml:lang="id">
      Manajer keuangan pribadi
    </p>
    <p xml:lang="it">
      Gestione finanziaria personale
    </p>
    <p xml:lang="ru">
      Персональный менеджер финансов
    </p>
  </description>
...
```

### CODE REVIEW

**Comment by nlogozzo** [comment]
There is a description string at the top of the Strings file:
```xml
<data name="Description" xml:space="preserve">
    <value>A personal finance manager.</value>
</data>
```

Can't we just use that string instead of making a new one for the same purpose?

File: NickvisionMoney.Shared/Resources/Strings.nl.resx [hunk 1]
```diff
@@ -525,10 +525,10 @@ Let op: niet-opgeslagen aanpassingen wordt gewist.</value>
...
 		<value>Money-rekening (*.nmoney)</value>
   </data>
+  <data name="MetaDescription.GTK" xml:space="preserve">
+		<value></value>
+  </data>
   <data name="Overview.Today" xml:space="preserve">
     <value>Dagoverzicht</value>
   </data>
-  <data name="DeleteTransaction.SourceRepeat" xml:space="preserve">
-    <value>Transactie verwijderen</value>
```
Comment (nlogozzo) [inline NickvisionMoney.Shared/Resources/Strings.nl.resx:532]: Why was this deleted?

**Review by nlogozzo — CHANGES_REQUESTED** [review]

**Comment by fsobolev** [comment]
> There is a description string at the top of the Strings file:
> 
> ```
> <data name="Description" xml:space="preserve">
>     <value>A personal finance manager.</value>
> </data>
> ```
> 
> Can't we just use that string instead of making a new one for the same purpose?

Then we need to remove "." in the end, GNOME HIG requires to not have it at desktop files comment and metainfo description.

File: NickvisionMoney.Shared/Resources/Strings.nl.resx (hunk 1 above)
Comment (fsobolev) [reply NickvisionMoney.Shared/Resources/Strings.nl.resx:532]: I didn't touch it, guess git decided to remove it without notifying me when I was resolving conflicts

**Comment by nlogozzo** [comment]
> Then we need to remove "." in the end, GNOME HIG requires to not have it at desktop files comment and metainfo description.

I'd be fine with that and then in the code we can just make it `mainWindowController.AppInfo.Description = $"{mainWindowController.Localizer["Description"]}.";` in GNOME/Program.cs and WinUI/App.xaml.cs

**Review by nlogozzo — APPROVED** [review]

**Comment by nlogozzo** [comment]
LGTM. Thanks for this :)
